### ISSUE TITLE
zamiana infinite scrolla na CTA

### ISSUE THREAD

**Issue by neu5** [neu5/zcyklu.pl#42]
@zcyklu, obecnie mamy infinite scrolla na stronie głównej zcyklu.pl
a od jakiegoś czasu zauważyłem tendencję żeby zamiast tego umieszczać CTA (call to action), czyli
przycisk, który trzeba kliknąć żeby wczytać więcej.
Chociażby github tak robi:
![image](https://user-images.githubusercontent.com/980955/30863451-61d01984-a2d1-11e7-8291-f080fcf59972.png)

Dzięki temu nie ma chociażby problemów z dostępem do stopki i łatwiej zapanować nad doczytywaniem potrzebnych rzeczy.

Co sądzisz o tym, żebyśmy poszli w te stronę? Dodanie przycisku np. "wczytaj kolejnych 16" czy coś takiego?

### Wymagania:
- [ ] ikonka ładowania po kliknięciu
- [ ] copy: "Mamy tego więcej!"
- [ ] graph: biały napis na czarnym prostokącie

**Comment by zcyklu** [comment]
Tak, dobry pomysł, bo uciekające stopki są wkurzające. Najlepiej jakiś tekst typu "więcej dobra" czy "więcej zcyklowego dobra" (skonsultuję to z "zarządem") na czarnym przycisku - jeśli na czarnym przycisku będą wysuwane cykle, bo tu bym chciała zachować konsekwencję w stylu.

**Comment by neu5** [comment]
Oki, to dawaj znać jak skonsultujesz to z górą 🗻

**Comment by zcyklu** [comment]
Ok, poproszę napis "Mamy tego więcej!". Najlepiej biały napis na czarnym prostokącie.

**Comment by neu5** [comment]
nie ma sensu robić zmian, które później i tak trzeba będzie poprawiać, dlatego
czeka na zrobienie #43
